### ISSUE TITLE
1.16.4 BungeeCord - Errors when running send and announce/bc commands and random drops from channels

### ISSUE THREAD

**Issue by Godlycreations** [kangarko/ChatControl#35]
Great Job on the updates!!!

I am seeing issues when running the commands '/ch s <channel> <message>' It sends the message to the channel and then sends an error message to the console. I have attached the error logs for you to review.

Also the command '/ch bc <message>' is not broadcasting to the other servers on the network. I have tried to use the '/chc a <message>' as well and both work fine for sending messages to everyone on the server that the command is run on, but neither go across the bungeecord to the other networks. I have to use the '/chc f <server> chc bc <message>' or '/chc f <server> chc a <message>' to make this happen. On a network with over 12 servers, that would be a very tedious task.

I have also noticed that at times, players get randomly dropped from channels. I have not set up any new channels yet, still playing with your default ones. I did not see any errors in chat other than at random times one of us would no longer be in the channel that we were just talking in.

I will keep playing around with the plugin and try setting up my chat channels to what the entire network is using.

Thanks for all that you are doing with this!

[elyon-error.log](https://github.com/kangarko/ChatControl-Red/files/5531747/elyon-error.log)
[es-error.log](https://github.com/kangarko/ChatControl-Red/files/5531748/es-error.log)

**Comment by kangarko** [comment]
Just to be clear are you using BungeeControl Red 3.0.2 and removed all else from BungeeCord?

**Comment by kangarko** [comment]
Just identified the first problem and solved it. Will see if I have time to push another release tonight. I am writing wiki right now

**Comment by Godlycreations** [comment]
Yes, I removed everything this morning and started with a fresh install on bungeecord and both servers. I wanted to test it as if it was never there before. So I deleted the MySql tables, all configs and jars before starting new install this morning.

**Comment by kangarko** [comment]
Okay so I identified and fixes the CHANNEL packet bug and a few others. There should be "@" messages in the console when players join, determining if they are going to be kicked or joined into channels. If you experience this weird drop again, please just open a new ticket, let me know as close as possible how to reproduce and I have a look.
